lib/graphql/schema/printer.rb

```diff
@@ -0,0 +1,131 @@
+module GraphQL
+  # Used to convert your {GraphQL::Schema} to a GraphQL schema string
+  #
+  # @example print your schema to standard output
+  #   Schema = GraphQL::Schema.new(query: QueryType)
+  #   puts GraphQL::Schema::Printer.print_schema(Schema)
+  #
+  module Schema::Printer
+    extend self
```

Comment:
This was just because there was no need to store state.  The private methods took a subset of the state, so using class methods makes it clearer than they don't use any implicit `self` state.  What you are suggesting is to instantiate an object just to pass arguments to a single private instance method.